dashboard/app/models/course_version.rb

```diff
@@ -58,14 +58,14 @@ def units
   # accessing them via course version. In the future, these fields will be moved
   # into the course version itself.
 
-  delegate :name, to: :content_root
-  delegate :pl_course?, to: :content_root
-  delegate :stable?, to: :content_root
-  delegate :launched?, to: :content_root
-  delegate :in_development?, to: :content_root
-  delegate :has_pilot_access?, to: :content_root
-  delegate :can_be_instructor?, to: :content_root
-  delegate :course_assignable?, to: :content_root
+  delegate :name, to: :content_root, allow_nil: true
+  delegate :pl_course?, to: :content_root, allow_nil: true
+  delegate :stable?, to: :content_root, allow_nil: true
+  delegate :launched?, to: :content_root, allow_nil: true
+  delegate :in_development?, to: :content_root, allow_nil: true
+  delegate :has_pilot_access?, to: :content_root, allow_nil: true
+  delegate :can_be_instructor?, to: :content_root, allow_nil: true
+  delegate :course_assignable?, to: :content_root, allow_nil: true
```

Comment:
The problem is that the CourseVersion exists but can not find the content_root it's supposed to point to. So when we call `course_version.has_pilot_access?` for example adding `&` would not fix it. We could go to writing out individual methods for each of the things that use delegate right now so that we could use `&`. Is that preferable?